syntax = "proto3";

import "mediocre/image/v1beta/image.proto";;
import "mediocre/options/v1beta/options.proto";

package mediocre.image.invert.v1beta;

service InvertService {
  rpc Invert(InvertRequest) returns (InvertResponse);
}

message InvertRequest {
  image.v1beta.Image image = 1;
  InvertParams params = 2;
}

message InvertParams {
}

message InvertResponse {
  image.v1beta.Image image = 1;
}
